workitem.2.go

```diff
@@ -165,3 +168,77 @@ func buildAbsoluteURL(req *goa.RequestData) string {
 	}
 	return fmt.Sprintf("%s://%s%s", scheme, req.Host, req.URL.Path)
 }
+
+// ConvertWorkItemToJSONAPI is responsible for converting given WorkItem model object into a
+// response resource object by jsonapi.org specifications
+func (c *Workitem2Controller) ConvertWorkItemToJSONAPI(ctx *app.UpdateWorkitem2Context, wi *app.WorkItem) *app.WorkItem2 {
```

Comment:
I highly suggest to pass in the second parameter as a value and not as a pointer: `wi app.WorkItem`. The reason is that I ran into a lot of troubles with the pointer that weren't obvious at first. For example when you create the `app.WorkItem2` and assign a pointer value from the `wi`.